### PR TITLE
Add ExteriorFloorSystemType Element

### PR DESCRIPTION
**Pull request by markborkum**
See `proposals/2019/Add ExteriorFloorSystemType.md` for details.

### CODE REVIEW

**Comment by markborkum** [comment]
@nllong, Step 4 of the implementation, which links exterior floors to premises, emulates the `auc:Foundations` child element of the `auc:Roof` element. An alternative approach is to emulate the `auc:SkylightIDs` child element of the `auc:RoofID` element.

File: proposals/2019/Add ExteriorFloorSystemType.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+   * `ExteriorFloorExteriorThermalAbsorptance`
+   * `InteriorVisibleAbsorptance`
+   * `ExteriorRoughness` &rarr; `auc:ExteriorRoughness`
+   * `Quantity` &rarr; `auc:Quantity`
+   * `YearInstalled` &rarr; `auc:YearInstalled`
+   * `UserDefinedFields` &rarr; `auc:UserDefinedFields`
+
+3. Add an `auc:ExteriorFloorSystems` element to the `auc:Systems` element.
+
+4. Add an `auc:ExteriorFloors` element to the `auc:Roof` element.
```
Comment (nllong) [inline proposals/2019/Add ExteriorFloorSystemType.md]: @markborkum, You mean `element to the auc:Section`, correct?

Comment (markborkum) [reply proposals/2019/Add ExteriorFloorSystemType.md]: @nllong Correct. Updated the proposal.

Comment (nllong) [reply proposals/2019/Add ExteriorFloorSystemType.md]: nvm. I see your comment above.

**Comment by markborkum** [comment]
@nllong Yes. I figured that [eventually] we'd need more than just R-values.

**Pull request merged**